tests/acceptance/network/test_network_actors.py

```diff
@@ -86,7 +86,6 @@ def test_treasure_map_cannot_be_duplicated(blockchain_ursulas, blockchain_alice,
     assert e.value.status == 402
 
 
-@pytest.mark.skip("See Issue #1075")  # TODO: Issue #1075
 def test_vladimir_illegal_interface_key_does_not_propagate(blockchain_ursulas):
```

Comment:
Glad to see good ol' Vlad coming back.

BTW, what's missing for closing #1075?

Reply:
Oh right - this closes it.